base/strings/util.jl

```diff
@@ -464,6 +464,55 @@ function hex2bytes(s::AbstractString)
 end
 
 """
+    hex2bytes(d::Vector{UInt8}, s::Vector{UInt8}, nInBytes::Int=length(s))
```

Comment:
`nInBytes` is a bit of weird capitalization. Perhaps use `ninbytes` or perhaps even better just `n`,